dashboard/app/views/levels/_project_ui.html.haml

```diff
@@ -23,6 +24,28 @@
     $('#save-project').show();
   }
 
+  var createApp = function(options) {
+    var defaultOptions = {
+      name: "Untitled",
+      createdAt: new Date,
+      updatedAt: new Date,
+      levelId: appOptions.levelId, // TODO this is not actually a unique identifier for the level
```

Comment:
Should we just add the fully-qualifited levelKey to appOptions?  I.e. what you'd pass into `Level#find_by_key`